src/Controller/ClippingController.php

```diff
@@ -47,7 +47,7 @@ public function indexAction(Request $request, EntityManagerInterface $em) {
         $qb->select('e')->from(Clipping::class, 'e')->orderBy('e.date', 'ASC');
         $query = $qb->getQuery();
 
-        $clippings = $this->paginator->paginate($query, $request->query->getint('page', 1), 25);
+        $clippings = $this->paginator->paginate($query, $request->query->getint('page', 1), 28);
```

Comment:
Is there a reason this one is 28 and the rest seem to be 24?

Reply:
Thanks for catching that! Switched to 24 in https://github.com/sfu-dhil/circus/pull/33/commits/a17474506705a252ef244a26664659b776025111